modules/nf-core/matrixfilter/main.nf

```diff
@@ -0,0 +1,22 @@
+process MATRIXFILTER {
+    tag "$meta"
+    label 'process_medium'
+    conda (params.enable_conda ? "r-base" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/r-base:4.2.1' :
+        'quay.io/biocontainers/r-base:4.2.1' }"
+
+    input:
+    tuple val(meta), path(samplesheet), path(abundance)
+
+    output:
+    tuple val(meta), path("*.filtered.tsv")             , emit: filtered
+    tuple val(meta), path("R_sessionInfo.log")          , emit: session_info
+    path "versions.yml"                                 , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
```

Comment:
How/where are the args passed to the script itself (I'm not familiar with templates admittedly)?

Reply:
Ah found it... maybe comment that `$task.ext.args` are specified on line XYZ in the scipt?

Reply:
Although, why not use the same template boilerplate to psas arguments?

```
    def args = task.ext.args ?: ''
    def prefix = task.ext.prefix ?: meta.id
    ```

Reply:
I _think_ I tried this and it didn't work in the template (newly defined variables weren't accessible in the template). But I'll double check.

Reply:
Explanatory note added